.prettierrc

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
I don't like this. If we have Eslint rules separated from Prettier, we can reach a loop of Prettier fixing a file and Eslint telling that is incorrect.

As the [eslint-plugin-prettier README](https://github.com/prettier/eslint-plugin-prettier#options) said, we can use `eslint --fix` to run Prettier through Eslint: 

> The rule is autofixable -- if you run eslint with the --fix flag, your code will be formatted according to prettier style.